content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,214 @@
+    tenantIdFromEnv: AAD_APP_TENANT_ID_ENV_VAR_NAME # Optional. You can use this instead of `tenantId` parameter.
+```
+
+This scaler is backed by Azure Data Explorer Goland SDK. Please see [this](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/api/golang/kusto-golang-client-library) page
+for further details.
+
+**Parameter list:**
+Provide either podIdentity or aad app creds (clientId, clientSecret and tenantId).
+- `clientId` - Id of the application from your Azure AD Application/service principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your service principal.
+- `clientSecret` - Password from your Azure AD Application/service principal.
+- `tenantId` - Id of the Azure Active Directory tenant. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to 
+retrieve your tenant id. 
+- `podIdentity` - [Azure Active Directory pod-managed identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) can be used. Only "azure" is supported. 
+- `databaseName` - The name of the Data Explorer Database to query. Use the [Azure Command Line Interface](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) to run `az kusto database list` to see a list of available databases in your cluster.
+- `endpoint` - The endpoint to query your Data Explorer Cluster. Follow [this](https://docs.microsoft.com/en-us/cli/azure/kusto/cluster?view=azure-cli-latest#az-kusto-cluster-show) link to get your cluster's endpoint.
+- `metricName` - Name to assign to the metric. (Optional, if not set KEDA will generate a name based on the database name)
+- `threshold` - Value that is used as a threshold to calculate # of pods for scale target.
+
+The authentication parameters could be provided using environmental variables, instead of setting them directly in metadata. Here is a list of parameters you can use to retrieve values from environment variables:
+
+- `tenantIdFromEnv` - An environmental variable name, that stores Azure Active Directory tenant id. Follow [this](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-show) link to retrieve your tenant id. (Optional)
+- `clientIdFromEnv` - An environmental variable name, that stores Application id from your Azure AD Application/service principal. Follow [this](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest) link to create your service principal. (Optional)
+- `clientSecretFromEnv` - An environmental variable name, that stores password from your Azure AD Application/service principal. (Optional)
+
+### Query Guidance
+
+It is important to design your query to return 1 row. A good practice is to add "| limit 1" at the end of your query.
+
+Data types of your query result should be: real, int or long. Other data types are not supported. Later, during runtime, your data will be converted to int64.
```

Comment:
```suggestion
The only supported data types for your query result are `real`, `int` or `long`. Later, during runtime, your data will be converted to int64.
```

Reply:
Is "Later, during runtime, your data will be converted to int64." really important to people? I don't think so but you might have a concern here?